cli/tools/publish/unfurl.rs

```diff
@@ -355,7 +355,7 @@ impl<TSys: FsMetadata + FsRead> SpecifierUnfurler<TSys> {
           ..
         } => match dep_result {
           Ok(dep) => match dep {
-            PackageJsonDepValue::File(_) => {
+            PackageJsonDepValue::File(_) | PackageJsonDepValue::JsrReq(_) => {
```

Comment:
Actually, probalby for all these cases we should surface an error for the time being.